src/chigame/games/models.py

```diff
@@ -6,13 +6,101 @@
 
 class Game(models.Model):
     """
-    A game (Chess, Checkers, etc.)
+    A game like Chess, Checkers, etc.
     """
 
+    # ================ BASIC INFORMATION ================
     name = models.TextField()
     description = models.TextField()
+    year_published = models.PositiveIntegerField(null=True)
+
+    # NOTE:
+    # Regular game images are not to be stored in the repository due to their large size.
+    # It is recommended to store them externally, for instance, on a dedicated server or a
+    # BLOB (Binary Large Object) storage service such as AWS S3.
+    image = models.URLField(default="/static/images/no_picture_available.png")
+
+    # ================ GAMEPLAY INFORMATION ================
+    rules = models.TextField(null=True)
+
     min_players = models.PositiveIntegerField()
     max_players = models.PositiveIntegerField()
+    suggested_age = models.PositiveSmallIntegerField(
+        null=True
+    )  # Minimum recommendable age. For example, 8+ would be stored as 8.
+
+    expected_playtime = models.PositiveIntegerField(null=True)  # In Minutes
+    min_playtime = models.PositiveIntegerField(null=True)
+    max_playtime = models.PositiveIntegerField(null=True)
+
+    complexity = models.PositiveSmallIntegerField(null=True)  # 1-5, 1 being the easiest
+    category = models.ManyToManyField("Category", related_name="games")
+    mechanics = models.ManyToManyField("Mechanic", related_name="games")
+
+    # ================ OTHER ================
```

Comment:
Appreciate the sub-divisions of fields. For a particularly important and large model like the game model this makes it significantly more readable. This is _especially true for those outside the team._